MyString.java

```diff
@@ -49,8 +75,16 @@ public static boolean subsetOf(String str1, String str2) {
      * @return a string consisting of the characters of str, separated by spaces.
      */
     public static String spacedString(String str) {
-        //// Replace the following statement with your code
-        return null;
+        String newword = "";
+        if(str.length() == 0){
+            return "";
+        }
+        for (int i = 0; i < str.length()-1; i++ ){
+            newword = newword + str.charAt(i) + " ";
+
+        }
+        newword = newword + str.charAt(str.length()-1);

```

Comment:
Better to put inside loop. 
see condition of last in recitation 4, array print function